domains/kunalpurandare.json

```diff
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "kunalpurandare"
+    },
+    "record": {
+        "CNAME": "https://kunalpurandare.github.io"
```

Comment:
Invalid CNAME. CNAME cannot contain protocols, file paths, URL parameters, or anything combination of those. See docs for more information: https://docs.is-a.dev/ Reopen PR when this is fixed.